crates/client-api/src/routes/database.rs

```diff
@@ -741,26 +561,26 @@ pub async fn delete_database<S: ControlStateDelegate>(
 }
 
 #[derive(Deserialize)]
-pub struct SetNameQueryParams {
-    domain: String,
-    database_identity: IdentityForUrl,
+pub struct AddNameParams {
+    // this field is named such because that's what the route is under.
+    // TODO: find out if there's a reason this can't be specified as a domain name
```

Comment:
What does this `TODO` comment mean? You should be able to set a new database name even if you use the old name to identify it.

Reply:
Alright, then I'll switch it over.